File: in_1.py
from pythonscript import js
js('var py = require("./py")')
js('console.log("Test in_1")')
def foo(x, y=1, **kwargs):
    print x+y
foo(**{'x': 1, 'y': 2})
js('console.log("End test in_1")')

File: js/in_1.js
py = require ("./py");
console.log ("Test in_1");
foo = py.def (py.kwargs, {"y":1}, function (x, y, kwargs) {
    console.log (x + y);
});
foo ([], {"x":1, "y":2});
console.log ("End test in_1");

